### DISCUSSION TITLE
Custom Piano Music

### DISCUSSION THREAD

**Discussion by FluffyMuffinMeyer** [Monika-After-Story/MonikaModDev#7786] [general - piano notes]
[tetris.txt](https://github.com/Monika-After-Story/MonikaModDev/files/2537017/tetris.txt)

Tetris theme!

only the first two lines, because the piano is not big enough to handle more of the song,
I couldn't quite figure out how the verse system works, so there needs to be a split added.
Also, the text is up in the air, you can have Moni say whatever you'd like.

Here are the keys, so you dont need to translate:

U R T Y T R E E T U Y T R T Y U T E E

Y I P O I U Y T R R T Y U T E E

**TETRIS WHEN POTATO?**

**Comment by ThePotatoGuy** [comment]
will look at this next week probably.

**Comment by ThePotatoGuy** [comment]
moving this to low priority. backlog of pull requests is more important rn.

**Comment by ThePotatoGuy** [comment]
[keeping this open so i remember it]

**Comment by zyroscope** [comment]
hey, i've finished fixed ur chord.
and this is the final result


U R T Y T R E E T U Y T R R T Y U T E E

Y Y I P O I U T U Y T R R T Y U T E E

Y Y I P O I U U T U Y T R R T Y U T E E

**Comment by ThePotatoGuy** [comment]
so we're changing how we approach piano jsons, and instead will handle these like spritepack content (aka external zip)
